CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 ## Code Contribution guidelines
 ### Dev Setup and debugging help
-Read the [FAQ on the Wiki](https://github.com/vmware/docker-volume-vsphere/wiki#faq)
+Read the [FAQ on the Wiki](https://github.com/vmware/vsphere-storage-for-docker/wiki#faq)
```

Comment:
Thanks for catching! no need for PR, I will update the title at github wiki page.